src/components/Modal/IdentityConnectModal/UnlinkAcountCard.tsx

```diff
@@ -31,24 +35,42 @@ function UnlinkAccountCard({
     }
   }
 
+  const handleUnlinkConfirmation = async () => {
+    await handleUnlinkAccount(account)
+    setIsConfirmationModalOpen(false)
+  }
+
   return (
-    <Card className="UnlinkAccountCard">
-      <div className="UnlinkAccountCard__AccountData">
-        <div className="UnlinkAccountCard__Title">
-          {account}
-          <CheckCircle size="16" />
-        </div>
-        <div className="UnlinkAccountCard__AccountDetails">
-          {t('modal.identity_setup.unlink_account_details', {
-            username: accountUsername || null,
-            date: Time(verificationDate).fromNow(),
-          })}
+    <div>
+      <Card className="UnlinkAccountCard">
+        <div className="UnlinkAccountCard__AccountData">
+          <div className="UnlinkAccountCard__Title">
+            {account}
+            <CheckCircle size="16" />
+          </div>
+          <div className="UnlinkAccountCard__AccountDetails">
+            {t('modal.identity_setup.unlink_account_details', {
+              username: accountUsername || null,
+              date: Time(verificationDate).fromNow(),
+            })}
+          </div>
         </div>
-      </div>
-      <Button basic onClick={() => handleUnlinkAccount(account)} className="UnlinkAccountCard__Action">
-        {t('modal.identity_setup.unlink_action')}
-      </Button>
-    </Card>
+        <Button basic onClick={() => setIsConfirmationModalOpen(true)} className="UnlinkAccountCard__Action">
+          {t('modal.identity_setup.unlink_action')}
+        </Button>
+      </Card>
+      <ConfirmationModal
+        isLoading={false}
```

Comment:
You can add a loading for the confirmation modal using the `handleUnlinkConfirmation` function.

```
  const handleUnlinkConfirmation = async () => {
    setLoadingUnlink(true)
    await handleUnlinkAccount(account)
    setIsConfirmationModalOpen(false)
    setLoadingUnlink(false)
  }
```